Source/Core/Core/Brawlback/BrawlbackUtility.cpp

```diff
@@ -5,95 +5,43 @@
 
 namespace Brawlback
 {
-
+    void Reset(RollbackInfo& _rollbackInfo)
```

Comment:
Is this function being used anywhere? I can't find any usages at a quick glance. 

If so I'd suggest a couple style changes:
- rename function to resetRollbackInfo
- remove underscore from arg name

Reply:
I addressed your style changes in a push below.